website/content/blog/Embracing-Edge-Computing-with-Kepler.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Embracing Edge Computing with Kepler - A Dive into Remote Monitoring, Centralized Dashboarding, and Visualization
+description: Kepler enables remote monitoring, centralized dashboarding, and visualization of energy consumption, revolutionizing energy observability at edge.
+slug: 2023-embracing-edge-computing-with-kepler
+weight: 1
+date:   2023-10-09 10:00:00 +0000
+author: Parul Singh
+---
+
```

Comment:
```suggestion
*Community guest post as part of the [cloud native sustainability week](https://tag-env-sustainability.cncf.io/cloud-native-sustainability-week/)*
```
On the TAG ENV blog we need to highlight if a blog is from the TAG directly (an artifact from a WG for example)